docs/src/pages/concepts/layouts.mdx

```diff
@@ -0,0 +1,45 @@
+
+## Layout Foundation
+
+Because layout patterns are such an essential part of building great applications, Marigold comes with a set of layout components that will cover almost, if not all, necessary patterns.
+
+The majority of layouts used on the web are not that unique. They often can be broken down into just a handful of layout patterns, which then can be combined to form the layout of a page.
+
+Each of Marigold's layout components is build to do one layout pattern well. Like all Marigold's component, they are composable so that the can be assembled until the desired layout is achieved. This makes it possible to build more complex layouts from just the given set of primitive layout components.
+
+## Layout Components
+
+The move to [component-based development](https://www.componentdriven.org/) has enabled a ton of incredible improvements when it comes to building digital user interfaces. It changed the way how we think about building applications entirely. In combination with the principles from [Atomic Design](https://atomicdesign.bradfrost.com/) we are no longer building applications page by page. Instead we are focusing on constructing applications by using reusable pieces of UI (a.k.a. components). This not only improves the overall user experience and quality, it also boosts the velocity and convenience of development with which digital products get build.
+
+A majority of the CSS layout patterns unfortunately predate this way of developing applications and as a result break the [modularity and composition assumptions of components](https://addyosmani.com/first/). CSS properties like `margin`, `flex-direction` or `align-self` that affect, or are affected by, elements outside of a components boundary will cause them to appear not as intended. Because of this, some people go as far as [considering margin as harmful](https://mxstbr.com/thoughts/margin/) and advice against its usage.
+
+> “Components shouldn't contain surrounding whitespace”
+>
+> — Mark Dalgleish, Rethinking Design Practices
+
+To avoid unexpected behavior and layout shifts, Marigold follows these the principles. Managing whitespace, as welle as positioning and arranging components is reserved for a dedicated set of layout components. This approach ensures that components are as composable as possible while keeping white space predictable.
+
+The components listed below can be nested within each other any number of times to create a wide variety of common layouts. Being familiar with these components is an essential part of working effectively with Marigold.
+
+- [Aside](../components/aside)
+- [Aspect](../components/aspect)
+- [Box](../components/box)
+- [Breakout](../components/breakout)
+- [Center](../components/center)
```

Comment:
Content is missing

Reply:
Moved, since it's a content component 😄

Maybe we also need too remove it?